controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -1024,6 +1043,15 @@ private DeleteScopeStatus reportDeleteScopeMetrics(String scope, DeleteScopeStat
         return status;
     }
 
+    private DeleteScopeRecursiveStatus.Status reportDeleteScopeRecursiveMetrics(String scope, DeleteScopeRecursiveStatus.Status status, Duration latency) {
+        if (status.equals(DeleteScopeRecursiveStatus.Status.SUCCESS)) {
```

Comment:
Metrics related to stream and kvt deletion also need to be updated here, right?

Reply:
Yes, this is required too. To keep it simple have logged a new issue and will be taking up there. In order to keep track, here's the issue:- https://github.com/pravega/pravega/issues/6456

Reply:
This cannot be done in a separate PR, as from a metrics perspective it will be a regression from current behavior of Controller.